upcase_booklist_app/static/main.css

```diff
@@ -233,3 +240,43 @@
     text-decoration: underline;
 }
 
+
+/* author.html */
+
+.container {
+    background-color: #ddd;
+    margin: 50px;
+    padding: 75px;
+}
+
+.author-name {
+    display: flex;
+    flex-direction: column;
+    padding-left: 5px;
+}
+
+.author_book {
+    display: flex;
+    flex-direction: row;
+}
+
+.book-title {
+    color: #990E16;
+    text-decoration: none;
+    font-size: 25px;
+    font-weight: bold;
+}
+
+.book-title a {
+    text-decoration: none;
+    color: #990E16;
+}
+
+.book-title a:hover {
+    color: #990E16;
+    text-decoration: underline;
+}
+
+.text {
```

Comment:
CSS looks good all around! Nice use of classes; you appear to know most of the most useful properties. 👏